settings.gradle.kts

```diff
@@ -1,18 +1,18 @@
 rootProject.name = "velocity-language-kotlin"
 
 pluginManagement {
-    repositories {
-        mavenCentral()
-        gradlePluginPortal()
-    }
+  repositories {
+    mavenCentral()
+    gradlePluginPortal()
+  }
 
-    plugins {
-        val kotlinVersion: String by settings
+  plugins {
+    val kotlinVersion: String by settings
 
-        kotlin("jvm") version kotlinVersion
-        kotlin("kapt") version kotlinVersion
-        kotlin("plugin.serialization") version kotlinVersion
+    kotlin("jvm") version kotlinVersion
+    kotlin("kapt") version kotlinVersion
+    kotlin("plugin.serialization") version kotlinVersion
```

Comment:
```suggestion
    kotlin("jvm").version(kotlinVersion)
    kotlin("kapt").version(kotlinVersion)
    kotlin("plugin.serialization").version(kotlinVersion)
```

Reply:
if the API provides infix functions, you should use them, not a call through a dot.

Reply:
Just because something is possible, doesn't mean it should be used. I personally think the normal dot notation is cleaner here.